### ISSUE TITLE
Broken layout / JS errors in PMA 5.2

### ISSUE THREAD

**Issue by onlime** [phpmyadmin/phpmyadmin#17522]
Since having upgraded to **phpMyAdmin 5.2.0**, I am experiencing the following problem:

Login screen looks fine, but once having successfully logged in, the whole layout is broken and nothing is clickable. Only the sidebar and header menu appears, but nothing is clickable and no content is shown. Looks like the following error (from console) breaks the execution client-side:

```
Uncaught SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
    success functions.js:4538
    jQuery 4
2 functions.js:4538:37
```

When debugging this, it looked like it died in `Functions.configGet` callback function when trying to retrieve `NavigationWidth` from `localStorage`. 

I have tested this both on `RELEASE_5_2_0` and current `master` branch (d0c37da), same problem. I have also cleared all caching, cookies and `localStorage`.

### To Reproduce

Install/upgrade phpMyAdmin to 5.2.0 like this:

```bash
# initial install
$ git clone https://github.com/phpmyadmin/phpmyadmin.git

$ cd phpmyadmin
$ git pull
$ git checkout RELEASE_5_2_0
$ composer install --no-dev
$ yarn install --production
```

Access PMA and login with an existing user, could even login with `root`. No content/data is shown. Try to click on "Databases" or any other menu item - nothing works.

### Screenshots

<img width="1507" alt="pma-520-bug" src="https://user-images.githubusercontent.com/2759561/168238711-a1c67d67-6eeb-46ce-a362-b4ae99459156.png">

### Server configuration

- Operating system: Debian Bullseye
- Web server: Apache 2.4.53
- Database version: Percona Server for MySQL 8.0.27 
- PHP version: PHP 8.0.18 / 7.4.29
- phpMyAdmin version: 5.2.0

### Client configuration

- Browser: Firefox / Chrome
- Operating system: macOS 12.3.1

**Comment by onlime** [comment]
after having completely removed `node_modules` and run `yarn install` again, the header/sidebar menu items are clickable again, but none of them works. Still no content is shown and not even the logout button works.

**Comment by MauricioFauth** [comment]
Hi, @onlime. Thank you for reporting this issue.
Could you please try deleting the `libraries/cache/routes.cache.php` file to see if that fixes the issue?

**Comment by onlime** [comment]
GREAT @MauricioFauth Thanks a lot!
This did the trick on all of my servers, running stable PMA 5.2.0 on PHP 8.0.

I put the removal of `libraries/cache/routes.cache.php` into my deployment (Ansible), in case this ever happens again. But just noticed, you already fixed it in #17524 - wonderful!
